dockerfile/anaconda-rpm/Dockerfile

```diff
@@ -32,6 +31,8 @@ COPY ["anaconda.spec.in", "requirements.txt", "/root/"]
 
 # Prepare environment and install build dependencies
 RUN set -ex; \
+  # Disable production repositories
+  dnf config-manager --disable baseos appstream crb; \
```

Comment:
The `crb` repository should be disabled by default so we probably don't need that. On the other hand, it won't hurt to be explicit so feel free to leave it here.